src/translator/text_processor.cpp

```diff
@@ -3,27 +3,85 @@
 #include <vector>
 
 #include "annotation.h"
+#include "common/cli_helper.h"
 #include "common/options.h"
 #include "data/types.h"
 #include "definitions.h"
 
 namespace marian {
 namespace bergamot {
 
+namespace {
+ug::ssplit::SentenceStream::splitmode string2splitmode(const std::string &m) {
+  typedef ug::ssplit::SentenceStream::splitmode splitmode;
+  if (m == "sentence") {
+    return splitmode::one_sentence_per_line;
+  } else if (m == "paragraph") {
+    return splitmode::one_paragraph_per_line;
+  } else if (m == "wrapped_text") {
+    return splitmode::wrapped_text;
+  } else {
+    ABORT("Unknown ssplitmode {}, Please choose one of {sentence,paragraph,wrapped_text}");
+  }
+}
+
+ug::ssplit::SentenceSplitter loadSplitter(const std::string &ssplitPrefixFile) {
+  // Temporarily supports empty, will be removed when mozilla passes ssplitPrefixFile
+  ug::ssplit::SentenceSplitter splitter;
+  if (ssplitPrefixFile.size()) {
+    std::string interpSsplitPrefixFile = marian::cli::interpolateEnvVars(ssplitPrefixFile);
+    LOG(info, "Loading protected prefixes for sentence splitting from {}", interpSsplitPrefixFile);
+    splitter.load(interpSsplitPrefixFile);
+  } else {
+    LOG(warn,
```

Comment:
What behaviour do we have in this case? What default is used?

Reply:
Regular expression based splitting without non-breaking prefixes accounted for. @ugermann can confirm.

Reply:
It breaks sentences where it shouldn't.